src/style.css

```diff
@@ -1 +1,76 @@
-/* add styles here */
+html {
+  margin: 0;
+}
+
+@font-face {
+  font-family: Avenir;
+  src: url(./fonts/Avenir-Book.ttf) format('ttf'),
+  url(./fonts/Avenir-Heavy.ttf) format('ttf');
+  font-weight: 300;
+}
+
+.search {
+  display: flex;
+  width: 100%;
+  align-items: center;
+  justify-content: center;
+  text-align: center;
+  border: 1px solid #E1E7ED;
```

Comment:
Move all repeatable colors to the variables.

[More info](https://developer.mozilla.org/en-US/docs/Web/CSS/Using_CSS_custom_properties)